src/eXpansion/Framework/GameTrackmania/DataProviders/Listener/LapDataListenerInterface.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace eXpansion\Framework\GameTrackmania\DataProviders\Listener;
+
+/**
+ * Class RaceDataListenerInterface
+ *
+ * @package eXpansion\Framework\GameTrackmania\DataProviders\Listener;
+ * @author  oliver de Cramer <oliverde8@gmail.com>
+ */
+interface LapDataListenerInterface
```

Comment:
ListenerInterfaceLapData same here, it should be prefixed...

Reply:
Right